codice/alliance
# CAL-370 Removed copy of log configuration files

#### What does this PR do?
This PR removes the copy of the log configuration files in Alliance and instead copies the ones from DDF.
#### Who is reviewing it? 
@peterhuffer @gordocanchola 
#### Choose 2 committers to review/merge the PR.
@bdeining
@clockard 
#### How should this be tested?
- Build Alliance, and confirm that the `org.ops4j.pax.logging.cfg` and `log4j2.config.xml` in etc/ include the same configuration as the 2 deleted files.
- For the `org.ops4j.pax.logging.cfg` configuration (used in Alliance by default), confirm that logging still works when Alliance is running. Confirm that the main log file is named `alliance.log`.
- Follow the Documentation (`Enabling Fallback Audit Logging`) to switch to use the `log4j2.config.xml`, and confirm that logging still works when Alliance is running. Confirm that the main log file is named `alliance.log`.
- Confirm that the system properties are set to the right value. You can use the `system:property` Karaf console command.

  | system property        | expected value           |
  | ------------- | ------------- |
  | org.codice.ddf.system.branding | alliance |
  | org.codice.ddf.system.siteName | alliance.distribution |
  | org.codice.ddf.system.version | 1.0.0-SNAPSHOT |
  | org.codice.ddf.system.organization | Codice |

#### Any background context you want to provide?
Is is a required hardening step to use the log4j2.config.xml configuration file.
#### What are the relevant tickets?
[CAL-370](https://codice.atlassian.net/browse/CAL-370)
#### Checklist:
